src/SourceBuild/tarball/content/repos/fsharp.proj

```diff
@@ -2,8 +2,10 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <BuildCommand>$(StandardSourceBuildCommand) $(StandardSourceBuildArgs)</BuildCommand>
+    <BuildCommandArgs>$(StandardSourceBuildArgs) /p:TreatWarningsAsErrors=false</BuildCommandArgs>
+    <BuildCommand>$(StandardSourceBuildCommand) $(BuildCommandArgs)</BuildCommand>
 
+    <NoWarn>$(NoWarn);NU5104</NoWarn>
```

Comment:
`AddCommonNoWarns` doesn't work for fsharp?

Reply:
An oversight.  Thanks for catching it.